source/jormungandr/jormungandr/scenarios/tests/journey_compare_tests.py

```diff
@@ -1151,3 +1151,44 @@ def test_activate_deactivate_min_taxi():
 
     f = jf.FilterTooShortHeavyJourneys(min_taxi=8, orig_modes=['taxi'], dest_modes=['taxi', 'walking'])
     assert not f.filter_func(journey)
+
+
+def test_filter_direct_path_mode_car():
+    # is_dp and not is_in_direct_path_mode_list
+    journey = response_pb2.Journey()
+    journey.tags.append("car")
+    journey.tags.append("non_pt")
+    f = jf.FilterDirectPathMode([])
```

Comment:
dp_mode is never gonna be empty as you test it before using it.
It's probably better to change this to the nominal case:
```suggestion
    f = jf.FilterDirectPathMode(["bike"])
```